omd.opam

```diff
@@ -1,24 +1,32 @@
 # This file is generated by dune, edit dune-project instead
```

Comment:
I didn't mean to commit this! Sorry :( 

Seems to indicate some inconsistency in our dune builds tho? I'm not sure what accounts for that.